test/src/api/responderTest.php

```diff
@@ -14923,10 +14953,137 @@ public function test_interface_game_046() {
             array('value' => 6, 'sides' => 12, 'recipe' => 's(12)', 'properties' => array('WasJustCaptured')),
             array('value' => 7, 'sides' => 20, 'recipe' => 'B(20)', 'properties' => array('WasJustCaptured')),
         );
-        # Bug #2040 - Vicious can no longer win at this point
+        $expData['playerDataArray'][0]['canStillWin'] = true;
+        $expData['playerDataArray'][1]['canStillWin'] = false;
 
         $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
-    }
+
+
+        ////////////////////
+        // Move 05 - responder004 surrenders
```

Comment:
I can't remember exactly, but it could have had to do with requiring the canStillWin to be NULL at game states outside of the round. Or perhaps it was something to do with Berserk and canStillWin. I'm happy to remove these unit tests if you'd like.